ecorise-react-app/src/Signup/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect, useRef } from 'react';
 import './index.css';
 import { FaUser, FaEnvelope, FaLock, FaEye, FaEyeSlash, FaPhone } from "react-icons/fa";
-import { signupUser } from '../utils/fetchUserprofiles';
+import { signupUser } from '../utils/postUserProfile';
```

Comment:
Your function name and file name must match. Does `postUserProfile` sound like `signUp`?